fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -272,7 +272,14 @@ def query_categories(category_keywords: Dict[str, List[str]] = KEYWORDS) -> pd.D
             # Raw messages
             kw_messages, error = query_keyword(keyword=k, anchor=last_msg_id)
             kw_messages = [{**x, **{'category': category, 'keyword': k}} for x in kw_messages]
-            new_messages += kw_messages
+            new_list = []
+            for x in kw_messages:
```

Comment:
I ran the code to double check and works fine! Thanks. I'm just going to push an update with a minor/later `todo` for us to consider, but it's really such an edge case that it's super low on the priority list and we don't have to worry about it right now.